src/include/windows/linux/nbd.h

```diff
@@ -0,0 +1,83 @@
+/* SPDX-License-Identifier: GPL-1.0+ WITH Linux-syscall-note */
```

Comment:
``win32`` is still commonly used to refer to the Windows API, regardless of the target platform [1]. We used ``windows`` in this case because the headers included here aren't part of the Windows API, but copying or emulating Linux or Posix ones, providing a compatibility layer.

[1] https://docs.microsoft.com/en-us/windows/win32/